fuse/test/file_lock_test.go

```diff
@@ -14,8 +14,7 @@ import (
 	"testing"
 )
 
-// See https://github.com/hanwen/go-fuse/issues/170
-func disabledTestFlock(t *testing.T) {
+func TestFlock(t *testing.T) {
```

Comment:
this test was a nice try, but I don't trust it. Can you make something that implements a lock, and verifies that the user-space locking code gets called?

Reply:
Fixed